src/SilentEcho.ts

```diff
@@ -7,8 +7,19 @@ export class SilentEcho {
         this.baseURL = "https://silentecho.bespoken.io/process";
     }
 
+    public normalizeMessage(message: string): string {
+        if (message === "no") {
```

Comment:
Do we want to add a trim() and toLowerCase() to this? Or have we aleady done that?

Reply:
We do only trim on script parsing on SilentEchoScript, but we should do trim and lower case here, good call!, addressed on: https://github.com/bespoken/silent-echo-sdk/pull/37/commits/6223628bee262caea515e15f54edfd592cf542eb